core/src/main/java/io/grpc/internal/AbstractClientStream.java

```diff
@@ -174,7 +174,7 @@ public final boolean isReady() {
     private ClientStreamListener listener;
 
     private boolean deframerClosed = false;
-    private Runnable deframerClosedTask;
+    protected Runnable deframerClosedTask;
```

Comment:
Did this slip in from an earlier review? You're right, but I don't think the commit where this was protected survives anymore - it was removed in the earlier rebase, and so this should no longer be an issue.